### PR TITLE
Hannacol/text based drills

### PR DESCRIPTION
**Pull request by hannacol**
Added logic for text based drills that do have inputs or outputted data.

- Removed buttons from drill index page as there are no leaderboards or stats associated with text drills
- Added screen for results page for text based drills to not display headers (may need reworked)
- Start drill takes you to a blank screen with submit drill (also may need reworked)
- Only display drills that are not "assignmentOnly" for players in DrillList

TODO:

- [x] Add checkmark on input screen instead of blank
- [x] Add big green checkmark on results screen

### CODE REVIEW

**Review by solderq35 — APPROVED** [review]

**Review by FrankreedX — CHANGES_REQUESTED** [review]
i dont like big if

**Review by FrankreedX — CHANGES_REQUESTED** [review]
Trying to go from Profile > IncreasedPressure shows this error
Screenshot 2024-06-03 at 4 07 37 PM

Maybe disable the View All Shots button in input.js
Screenshot 2024-06-03 at 4 02 50 PM

Lastly, Make the "completed" checkbox more consistent with normal drill inputs
Screenshot 2024-06-03 at 4 05 19 PM

**Comment by FrankreedX** [comment]
There are only 2 more things I want to request from this. It looks good otherwise.
First, hasStats = false drills shouldn't be visible in Team > Profile.
Second, in coach, completed assignment of drill with hasStats = false shouldn't be clickable.

**Review by FrankreedX — CHANGES_REQUESTED** [review]

File: app/segments/drill/[id]/submission/input.js [hunk 1]
```diff
@@ -119,17 +124,64 @@ async function uploadAttempt(
...
+      userId,
+      assignedTime,
+      drillId,
+      newAttemptRef.id,
+      currentTeamId,
+    );
+  }
+}
+
+async function uploadTextDrill(userId, assignedTime, drillId, currentTeamId) {
```
[the remark this answers is not in the document]
Comment (hannacol) [reply app/segments/drill/[id]/submission/input.js:149]: Forgot to remove that. Thanks

**Review by FrankreedX — APPROVED** [review]
nice!

**Pull request merged**